eZ/Publish/Core/FieldType/Image/ImageStorage/Gateway/DoctrineStorage.php

```diff
@@ -456,4 +456,38 @@ public function updateImagePath(int $fieldId, string $oldPath, string $newPath):
             ->execute()
         ;
     }
+
+    /**
+     * @throws \eZ\Publish\Core\IO\Exception\InvalidBinaryFileIdException
+     * @throws \Doctrine\DBAL\Driver\Exception
+     * @throws \Doctrine\DBAL\Exception
+     */
+    public function getImageReference(string $uri, int $fieldId): bool
+    {
+        $path = $this->redecorator->redecorateFromSource($uri);
+
+        $selectQuery = $this->connection->createQueryBuilder();
+        $selectQuery
+            ->select(1)
+            ->from($this->connection->quoteIdentifier(self::IMAGE_FILE_TABLE))
+            ->where(
+                $selectQuery->expr()->eq(
+                    $this->connection->quoteIdentifier('filepath'),
+                    ':path'
+                )
+            )
+            ->andWhere(
+                $selectQuery->expr()->eq(
+                    $this->connection->quoteIdentifier('contentobject_attribute_id'),
+                    ':fieldId'
```

Comment:
let's use here Doctrine naming convention. Whenever we use camel case for QB parameter names, it's technical debt.
```suggestion
                    ':field_id'
```